packages/seducer/src/logger.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-use-before-define */
+/**
+ * Summary: Prints the current processing action via the dev console, with it's previous state and the next state after passing inside the reducer.
+ *
+ * @param {{unknown}}           prevState           The current state of the store
+ * @param {{unknown}}           nextState           The next state after being pass through the reducer
+ * @param {{ type, payload }}   action              The actions that has being dispatched by the consumer
+ * @param {boolean}             isFromInterceptor   Let the consoler knowns if the actions is an original or has been override by the interceptors
+ *
+ * @return {{unknown}}          The next state
+ */
```

Comment:
Nice! So maybe we can add this package as excluded packages in `typesFiles.js` and `readmeFiles.js`

Reply:
it will not need it, it will not find a React component in the `index.js` and will skip it.